packages/storage/src/userStorage.spec.ts

```diff
@@ -26,6 +27,14 @@ const initStubbedStorage = (): { storage: UserStorage; mockBuckets: Buckets } =>
     }),
   );
 
+  // const mockMetadataStore: UserMetadataStore = mock();
```

Comment:
Leftover comments?

Reply:
There is currently an issue with mocking UserMetadataStore. It just stops the process abruptly. I'm yet to figure it out, so I left this comment here, so It can be fixed later.